com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -348,11 +351,33 @@ private void Initialize(bool server)
 
             SceneManager = new NetworkSceneManager(this);
 
+            m_RpcBatcher = new RpcBatcher(this);
+
+            if (MessageHandler == null)
+            {
+                IInternalMessageHandler messageHandler = new InternalMessageHandler(this);
+
+#if DEVELOPMENT_BUILD || UNITY_EDITOR
+                messageHandler = new InternalMessageHandlerProfilingDecorator(messageHandler);
+#endif
+                // Only create this if it's not already set (like in test cases)
+                MessageHandler = messageHandler;
+            }
+
```

Comment:
@bendoyon Can you double check what's going on here? In meeting you mentioned that an identical change was merged into dev